app/controllers/questions_controller.rb

```diff
@@ -128,6 +128,17 @@ def build_csv
     end
   end
 
+  def import
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def import_csv
+    Question.import(params[:file], current_user)
```

Comment:
Right now, if there's something wrong with the CSV, the app just crashes. Maybe have a simple try/catch, and set the notice to "Upload failed—bad CSV" or something.